docs/doxygen/mainpages/const_cpp.h

```diff
@@ -26,10 +26,11 @@ using @ifdef_ and not @if_.
 @itemdef{__WXBASE__, Only wxBase, no GUI features (same as @c wxUSE_GUI == 0)}
 @itemdef{__WXDFB__, wxUniversal using DirectFB}
 @itemdef{__WXGTK__, GTK+}
-@itemdef{__WXGTK12__, GTK+ 1.2 or higher}
+@itemdef{__WXGTK127__, GTK+ 1.2.7 or higher}
 @itemdef{__WXGTK20__, GTK+ 2.0 or higher}
```

Comment:
I wonder if we should replace "2.0" with "2.6" here, as we don't support versions < 2.6.

Reply:
I would guess adding an note about the lowest version supported is 2.6 would be better.
How about  "GTK+ 2.0 (2.6) or higher"?

Reply:
Looks good, thanks!

Reply:
Added changes